README.md

```diff
@@ -13,19 +13,21 @@ Take a look at the [concepts](/docs/concepts/README.md) page for a brief descrip
 ## Conceptual Diagram
 <img src="site/static/images/jobset_diagram.png" alt="jobset diagram">
 
-## Installation
+## Features overview
 
-**Requires Kubernetes 1.26 or newer**.
+- **Support for multi-template jobs**: JobSet models a distributed training workload as a group of K8s Jobs. This allows a user to easily specify different pod templates for different distinct groups of pods (e.g. a leader, workers, parameter servers, etc.), something which cannot be done by a single Job.
 
-To install the latest release of JobSet in your cluster, run the following command:
+- **Automatic headless service configuration and lifecycle management**: ML and HPC frameworks require a stable network endpoint for each worker in the distributed workload, and since pod IPs are dynamically assigned and can change between restarts, stable pod hostnames are required for distributed training on k8s, By default, JobSet uses [IndexedJobs](https://kubernetes.io/blog/2021/04/19/introducing-indexed-jobs/) to establish stable pod hostnames, and does automatic configuration and lifecycle management of the headless service to trigger DNS record creations and establish network connectivity via pod hostnames.
 
-```shell
-kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/v0.4.0/manifests.yaml
-```
+- **Configurable success policies**: JobSet has [configurable success policies](https://github.com/kubernetes-sigs/jobset/blob/1ae6c0c039c21d29083de38ae70d13c2c8ec613f/examples/simple/success-policy.yaml) which target specific ReplicatedJobs, with operators to target `Any` or `All` of their child jobs. For example, you can configure the JobSet to be marked complete if and only if all pods that are part of the “worker” ReplicatedJob are completed. This enables users to use their compute resources more efficiently, allowing a workload to be declared successful and release the resources for the next workload more quickly.
```

Comment:
Should we use a perm link? 

A user could see a different example than what is in the repo.

Reply:
The configurable success example link is a permalink to a specific commit.

Reply:
I see, so you think for these examples we should link to main?

Reply:
Updated links to use v0.4.0 for example yamls for now.

We can think about having examples directly in the docsite separately, for now I just want the feature overview list on the Github README.